packages/perseus/src/widgets/input-number.tsx

```diff
@@ -202,13 +202,6 @@ class InputNumber extends React.Component<Props> {
 
     _handleFocus: () => void = () => {
         this.props.onFocus([]);
```

Comment:
The interaction callback is no longer necessary here as we're calling it in the server-item-renderer now.